app/src/main/java/it/kapfer/digitalesamt/liberator/ModuleMain.kt

```diff
@@ -69,6 +69,7 @@ class ModuleMain : IXposedHookZygoteInit, IXposedHookLoadPackage {
         XposedBridge.log("Hooking DeviceIntegrityCheck")
         XposedHelpers.findAndHookMethod(DEVICE_INTEGRITY_CHECK_CLASS, lpparam.classLoader, "checkIntegrity", XC_MethodReplacement.DO_NOTHING)
         XposedHelpers.findAndHookMethod(DEVICE_INTEGRITY_CHECK_CLASS, lpparam.classLoader, "checkIntegrityForceCheck", XC_MethodReplacement.DO_NOTHING)
+        XposedHelpers.findAndHookMethod(DEVICE_INTEGRITY_CHECK_CLASS, lpparam.classLoader, "attestationSupportCheck", XC_MethodReplacement.DO_NOTHING)
```

Comment:
Could you please add a version check to that again? `findAndHookMethod` throws an exception when it doesn't find the requested method to hook.
Please also consider that both the _Serviceportal Bund_ and the _Digitales Amt_ app share this method - is this method used for the second app as well? If yes, which version do we have to consider there? It might be necessary to split this into two methods.

Reply:
Hi, the current version SPB 1.7.5 also uses this method. I didn't fit a version check exactly for this reason. I mean I could split it into 2 Methods for the apps, but it works for lower versions (if you don't mind the exception :) )

Reply:
I should probably split it tho...